common/device-model.cpp

```diff
@@ -1433,6 +1433,13 @@ namespace rs2
                         }
                     }
                 }
+
+                for( auto & sub : subdevices )
```

Comment:
When you open HDR configuration window than, while it stays open, you select the menu the config window gets messed up. All open items are dark (like disabled), new items open up, and they are connected. Minimizing the old "Preset Item 1" also minimizes the new and so on...

I say when HDR config window is open allow interactions only with it. User must "Apply" or "Cancel" before doing other things in the viewer.